auth-web/src/components/auth/TestimonialQuotes.vue

```diff
@@ -45,5 +69,36 @@ export default class TestimonialQuotes extends Vue {}
       height: 150px;
       width: 50%;
     }
+
+    @keyframes fading {
+      90% {
+        opacity: 1;
+      }
+      100% {
+        opacity: 0;
+      }
+      0% {
+        opacity: 0;
+      }
+      10% {
+        opacity: 1;
+      }
+    }
+
+    .fading {
+      animation: fading 5s ease-in-out infinite;
+    }
+  }
+
+  #testimonials-container::after {
+    border-color: transparent #003366 transparent transparent;
```

Comment:
can the color codes be replaced with the scss theme variables?

Reply:
Absolutely, will update!